src/themes/uclouvain/app/entity-groups/publication-entity/search-result-list-elements/publication-search-result/publication-search-result-list-element.component.scss

```diff
@@ -0,0 +1,10 @@
+.search-result {
+    box-shadow: rgba(0, 0, 0, 0.03) 0px 6px 24px 0px, rgba(0, 0, 0, 0.08) 0px 0px 0px 1px;
```

Comment:
be carefull with box-shadow in professional context, it could look "cheap" very quickly